--- src/org/jaxb/gpxbind/GpxType.java	2016-02-03 01:22:32.302548773 +0200
+++ ../src/org/jaxb/gpxbind/GpxType.java	2016-02-03 01:35:11.370558609 +0200
@@ -14,6 +14,7 @@
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlAttribute;
 import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.*;
 
 
 /**
@@ -46,6 +47,8 @@
  * 
  * 
  */
+
+
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "gpxType", propOrder = {
     "metadata",
@@ -54,6 +57,7 @@
     "trk",
     "extensions"
 })
+@XmlRootElement(name = "gpx")
 public class GpxType {
 
     protected MetadataType metadata;
